client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -65,18 +67,24 @@ export const RequestLineEditFormLayout = ({
       paddingBottom={1}
       boxShadow={theme => theme.shadows[2]}
     >
-      <Grid item xs={4}>
-        {Left}
-      </Grid>
-      <Grid
-        item
-        xs={4}
-        paddingBottom={1}
-        display="flex"
-        flexDirection="column"
-        justifyContent="flex-end"
-      >
-        {Middle}
+      <Grid item xs={8} direction="column" justifyContent="space-between">
+        <Grid item xs={12}>
+          {Top}
+        </Grid>
+        <Grid
+          item
+          xs={12}
+          container
+          direction="row"
+          justifyContent="space-between"
+        >
+          <Grid item xs={6}>
+            {Left}
+          </Grid>
+          <Grid item xs={6}>
+            {Middle}
+          </Grid>
+        </Grid>
```

Comment:
Maybe this is okay - but felt a bit janky and fragile.

Reply:
I think it's ok - it was a little janky to begin with.
Don't think that it's fragile - though it does mess with the `left`, `right` etc now that there's a `top` in the mix..

The form is looking a little cramped now - I wondered about lining up the two stacks of label:value pairs like this:

<img width="1069" alt="image" src="https://github.com/openmsupply/open-msupply/assets/9192912/efdeebe8-c61e-4840-95ad-7363e882c758">

by changing the left bit:

```
<Grid
            item
            xs={6}
            flexDirection="column"
            display="flex"
            justifyContent="flex-end"
          >
```